docs/src/single-pool.mdx

```diff
@@ -0,0 +1,477 @@
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool initialize Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const voteAccountAddress = new PublicKey('Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC');
+const transaction = await SinglePoolProgram.initialize(
+  connection,
+  voteAccountAddress,
+  feePayerAddress,
+);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const voteAccountAddress = 'Ammgaa2iZfA745BmZMhkcS27uh87fEVDC6Gm2RXz5hrC' as VoteAccountAddress;
+const transaction = await SinglePoolProgram.initialize(
+  rpc,
+  voteAccountAddress,
+  feePayerAddress,
+);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+</Tabs>
+
+#### Managing token metadata
+
+By default, when a pool is created, it also creates Metaplex token metadata for the mint associated with the pool.
+If, for whatever reason, this was opted out of by the pool creator, anyone may create the default metadata permissionlessly:
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool create-token-metadata --pool DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const transaction = await SinglePoolProgram.createTokenMetadata(poolAddress, feePayerAddress);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const transaction = await SinglePoolProgram.createTokenMetadata(poolAddress, feePayerAddress);
+
+// sign with the fee payer
+```
+
+  </TabItem>
+</Tabs>
+
+The default token metadata is only minimally helpful, spotlighting the address of the pool.
+The owner of the vote account, however, can change the metadata to anything they wish.
+They prove their identity by signing with the vote account's authorized withdrawer;
+this is the only permissioned instruction on the pool.
+
+<Tabs className="unique-tabs" groupId="language-selection">
+  <TabItem value="cli" label="CLI" default>
+
+```console
+spl-single-pool update-token-metadata DkE6XFGbqSyYzRugLVSmmB42F9BQZ7mZU837e2Cti7kb "My Cool Pool" cPool
+```
+
+  </TabItem>
+
+  <TabItem value="jsc" label="WEB3.JS CLASSIC">
+
+```typescript
+const transaction = await SinglePoolProgram.updateTokenMetadata(
+  voteAccountAddress,
+  authorizedWithdrawerAddress,
+  'My Cool Pool',
+  'cPool',
+);
+
+// sign with the fee payer and authorized withdrawer
+```
+
+  </TabItem>
+
+  <TabItem value="jsn" label="WEB3.JS NEXT">
+
+```typescript
+const transaction = await SinglePoolProgram.updateTokenMetadata(
+  voteAccountAddress,
+  authorizedWithdrawerAddress,
+  'My Cool Pool',
+  'cPool',
+);
+
+// sign with the fee payer and authorized withdrawer
+```
+
+  </TabItem>
+</Tabs>
+
+A URL may also be included as an optional parameter.
```

Comment:
Could you include the URL in the examples? I think most people will want to do that